tests/migration/ember-addon/steps/analyze-addon/customizations.test.js

```diff
@@ -63,17 +62,5 @@ test('migration | ember-addon | steps | analyze-addon > base case', function ()
       'modifiers/container-query.js',
       'template-registry.js',
     ],
-  };
-
-  loadFixture(inputProject, options);
-
-  // Run the step
-  const augmentedOptions = {
-    projectRoot: 'tmp/ember-container-query-typescript',
-  };
-
-  assert.deepEqual(analyzeAddon(augmentedOptions), expectedValue);
-
-  // Check idempotence
-  assert.deepEqual(analyzeAddon(augmentedOptions), expectedValue);
```

Comment:
After merging #3, I realized that it doesn't make sense to check idempotence. After all, the codemod makes an early exit when an addon is no longer v1.

https://github.com/ijlee2/ember-codemod-v1-to-v2/blob/0.1.2/src/migration/ember-addon/index.js#L19-L22